templates/page.html

```diff
@@ -81,7 +81,7 @@ <h1 class="article-title">
             {%- endif -%}
 
             {% if page.updated %}
-                </ul><ul class="meta last-updated"><li>{{ macros_translate::translate(key="last_updated_on", default="Last updated on", language_strings=language_strings) }} {{ macros_format_date::format_date(date=page.updated, short=true, language_strings=language_strings) }}</li>
+                </ul><ul class="meta last-updated"><li>{{ macros_translate::translate(key="last_updated_on", default="Last updated on", language_strings=language_strings) }} {{ macros_format_date::format_date(date=page.updated, short=false, language_strings=language_strings) }}</li>
```

Comment:
❌

Reply:
In my opinion, it would be clearer to include the year, just a recommendation. I appreciate your work.